projects/sqlalchemy-utils/fuzz_type.py

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/python3
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import atheris
+import sys
+
+with atheris.instrument_imports():
+   import sqlalchemy
+   from sqlalchemy import create_engine, Integer, String, MetaData
+   from sqlalchemy import Table, Column, Sequence 
+   from sqlalchemy.sql import text, select
+   from sqlalchemy.exc import SQLAlchemyError
+   from sqlalchemy_utils import ArrowType, ChoiceType, ColorType
+   from sqlalchemy_utils import CountryType, Country, EmailType, JSONType
+   from sqlalchemy_utils import IPAddressType, ScalarListType
+   from sqlalchemy_utils import URLType, UUIDType, WeekDays, WeekDaysType 
```

Comment:
The imports are very messy throughout the fuzzers. Please clean this up

Reply:
I pushed up a few changes to clean this up -- also realised the `babel` imports were needed for pyinstaller